core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -801,8 +801,10 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     awaitAssignment(consumer, partitions.toSet)
 
     val producer = createProducer()
-    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = 15, _))
-    val consumerRecords = consumeRecords(consumer, producerRecords.size)
+    // we produce 10 records for each topic partition. There are 3 topics, and 30 partitions each topic,
+    // so total producerRecords size should be 10 * 3 * 30 = 900
+    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = 10, _))
+    val consumerRecords = consumeRecords(consumer, producerRecords.size, waitTimeMs = 90 * 1000)
```

Comment:
Personally, 90 seconds is too long to be a test case. Reducing the produce size can't resolve this issue?

Reply:
Agree! I increase to 90 secs just in case. I think reduce the record size is good enough.

Reply:
revert back to 60 secs now.